src/gluonts/dataset/loader.py

```diff
@@ -14,19 +14,14 @@
 # Standard library imports
 import functools
 import itertools
-from typing import (
-    Any,
-    Dict,
-    Iterable,
-    Iterator,
-)
+from typing import Any, Dict, Iterable, Iterator, List, Union
```

Comment:
Are you using `List` and `Union`?

Reply:
Thanks! removing them. Was before the `ContextType` refactoring.